libfwupdplugin/fu-hid-device.c

```diff
@@ -193,6 +193,14 @@ fu_hid_device_open(FuDevice *device, GError **error)
 	FuHidDevicePrivate *priv = GET_PRIVATE(self);
 	FuUsbDeviceClaimFlags flags = 0;
 
+	if (fu_device_has_private_flag(device, FU_DEVICE_PRIVATE_FLAG_NO_OPEN_FOR_PROBE)) {
+		g_set_error(error,
```

Comment:
Ohh if we got to ->open() I think we're checking far too late -- can we do this check in ->probe()?